HANDBOOK.md

```diff
@@ -1285,6 +1286,30 @@ targets = {
 Requires:
 * gobject-introspection Python bindings
 
+### `launch`
+
+The `launch` target excutes the specified program and its arguments. It is similar
+to `pipe` but it doesn't open a pipe to the program. It provides stdout as response
+to configured queue.
+Example use cases are f.e. IoT buttons which publish a message when they are pushed
+and the excute an external program. It is also a clone of [mqtt-launcher](https://github.com/jpmens/mqtt-launcher).
+
+```ini
+[config:launch]
+targets = {
+              # full_topic, topic[0], topic[1], args[0], .....
+   'touch'    : [ None,0,0,'touch', '/tmp/executed' ],
+   'fritzctl' : [ None,0,0,'/usr/bin/fritzctl', 'temperature', "{args[0]}", "{args[1]}" ]
+   'backup'   : ["response/{topic[1]}/{topic[2]}",0,0,'/usr/bin/sudo','/usr/sbin/dirvish','--vault', "{args[0]}" ],
+   }
```

Comment:
I think it would be very useful if you could add a short explanation in a comment on what each of these targets accomplish, maybe in a `# line` before the individual target?